app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,323 @@
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/start_end_margin"
+            app:layout_constraintBottom_toBottomOf="@+id/tv_reviews_number"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/tv_reviews_number"
+            app:srcCompat="@drawable/star_icon_full"
+            tools:ignore="ContentDescription" />
+
+        <ImageView
+            android:id="@+id/img_star2"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/star_margin"
+            app:layout_constraintBottom_toBottomOf="@+id/img_star1"
+            app:layout_constraintStart_toEndOf="@id/img_star1"
+            app:layout_constraintTop_toTopOf="@+id/img_star1"
+            app:srcCompat="@drawable/star_icon_full"
+            tools:ignore="ContentDescription" />
+
+        <ImageView
+            android:id="@+id/img_star3"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/star_margin"
+            app:layout_constraintBottom_toBottomOf="@+id/img_star1"
+            app:layout_constraintStart_toEndOf="@id/img_star2"
+            app:layout_constraintTop_toTopOf="@+id/img_star1"
+            app:srcCompat="@drawable/star_icon_full"
+            tools:ignore="ContentDescription" />
+
+        <ImageView
+            android:id="@+id/img_star4"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/star_margin"
+            app:layout_constraintBottom_toBottomOf="@+id/img_star1"
+            app:layout_constraintStart_toEndOf="@id/img_star3"
+            app:layout_constraintTop_toTopOf="@+id/img_star1"
+            app:srcCompat="@drawable/star_icon_full"
+            tools:ignore="ContentDescription" />
+
+        <ImageView
+            android:id="@+id/img_star5"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/star_margin"
+            app:layout_constraintBottom_toBottomOf="@+id/img_star1"
+            app:layout_constraintStart_toEndOf="@id/img_star4"
+            app:layout_constraintTop_toTopOf="@+id/img_star1"
+            app:srcCompat="@drawable/star_icon_empty"
+            tools:ignore="ContentDescription" />
+
+        <TextView
+            android:id="@+id/tv_reviews_number"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/start_end_margin"
+            android:layout_marginTop="4dp"
+            android:text="@string/reviews_number"
+            android:textColor="#6D6D80"
+            android:textSize="14sp"
```

Comment:
move to res/dimens :) well, you've got the point :D

Reply:
> move to res/dimens :) well, you've got the point :D

Got the point :) Fixed what was found)